### PR TITLE
Add VS4Mac/macOS pivot to MAUI-Blazor tutorial

### PR DESCRIPTION
**Pull request by guardrex**
Addresses #26364

[Internal Review Topic](https://review.learn.microsoft.com/en-us/aspnet/core/blazor/hybrid/tutorials/maui?view=aspnetcore-7.0&branch=pr-en-us-27678)

* This is in pretty much 🦖 ***Rex Pea-brain Dump***&trade; 🦖 state due to the speed that I cranked it out this morning 🏃‍♂️. It's a bit rough, but I think it's ok to take feedback on it in this state given that I'm about out of 🕦 this week. I'll try to address feedback today (Friday) if it arrives. I'll review/edit/polish on Monday morning either way.
* Check the pivot selector prompt text ("**Choose a Visual Studio operating system**", `aspnetcore/zone-pivot-groups.yml`) to see what you want that to say.
* I placed an IMPORTANT note about the IDE install order and what to do if it's accidentally performed in reverse. ***I did it in reverse by mistake*** 🙈 and hit the MIA SDK error, so I'd like to head off doc repo issues by leaving this guidance in here *somewhere*. It could go at the end with a cross-link from the top or in the MAUI tutorial overview topic with a cross-link from the top ... or where I have it now on the PR.
* I had a warning here about the Java SDK, but it didn't affect this scenario. I haven't researched it, so idk what's what with it.

  warning

* WRT the WinOS pivot: I sent the remarks about the WinOS problems in the offline chat. I think that the **Dock** > **Fill** problem was present back in the 6.0 release, too. The component reference problem goes away after the app is built the first time.

### CODE REVIEW

File: aspnetcore/blazor/hybrid/tutorials/maui.md [hunk 1]
```diff
@@ -114,36 +122,163 @@ Close the **Android Device Manager** window. Wait until the emulated phone windo
...
+
+Name the project in the **Project name** field and select the **Create** button. The project name for this demonstration is `MauiBlazor`:
+
+:::image type="content" source="maui/_static/mac/step04.png" alt-text="Configure your new .NET MAUI Blazor App dialog with an project name of MauiBlazor.":::
+
+## Run the app on macOS with Mac Catalyst
+
+Select the play button (▶) to build and run the app:
+
+:::image type="content" source="maui/_static/mac/step05.png" alt-text="MauiBlazor window indicating the location of the play button.":::
```
Comment (danroth27) [inline aspnetcore/blazor/hybrid/tutorials/maui.md:212]: This screenshot shows a red squiggle under the component in the editor. Can we get a screenshot without that? Or is the Razor tooling in VS4Mac for .NET MAUI Blazor apps broken?

**Comment by danroth27** [comment]
> Check the pivot selector prompt text ("Choose a Visual Studio operating system", aspnetcore/zone-pivot-groups.yml) to see what you want that to say.

The current text seems fine.

> I placed an IMPORTANT note about the IDE install order and what to do if it's accidentally performed in reverse. I did it in reverse by mistake 🙈 and hit the MIA SDK error, so I'd like to head off doc repo issues by leaving this guidance in here somewhere. It could go at the end with a cross-link from the top or in the MAUI tutorial overview topic with a cross-link from the top ... or where I have it now on the PR.

The note and its location look reasonable to me.

> I had a warning here about the Java SDK, but it didn't affect this scenario. I haven't researched it, so idk what's what with it.

Sounds like something we should follow up on, but it doesn't block this doc. Can you create a VS feedback issue for this and share the link with us?

> WRT the WinOS pivot: I sent the remarks about the WinOS problems in the offline chat. I think that the Dock > Fill problem was present back in the 6.0 release, too. The component reference problem goes away after the app is built the first time.

These are problems with the Windows Forms experience, right? I'll follow up with you on these separately.

**Comment by guardrex** [comment]
> These are problems with the Windows Forms experience, right? I'll follow up with you on these separately.

Yes, the screenshots for those two are in that email that you responded to yesterday. The subject line is "**Almost there on Blazor docs 7.0 coverage**". It's the first message of that chain.

> Can you create a VS feedback issue for this and share the link with us?

Yes. Will do. 👍 

> I'll need to get back to you tho in a few hours on getting back on the MacBook.

I'll get back to you in a few hours on that, too.

File: aspnetcore/blazor/hybrid/tutorials/maui.md (hunk 1 above)
Comment (guardrex) [reply aspnetcore/blazor/hybrid/tutorials/maui.md:212]: ***Confirmed!*** The red squiggle is still there in the UI. I'm opening a **Report a Problem** issue now from the MacBook. I'll post the link to it here when they send it.

As for the image here, I'll Photoshop that squiggle out 💪.

Comment (guardrex) [reply aspnetcore/blazor/hybrid/tutorials/maui.md:212]: Here's the Developer Community report on this one ...

https://developercommunity.visualstudio.com/t/Cant-find-SurveyPrompt-component-in-a-/10206835

**Comment by guardrex** [comment]
@danroth27 ...

* The image is updated ... ***NO*** squiggle!
* The Dev Community report is at https://developercommunity.visualstudio.com/t/Cant-find-SurveyPrompt-component-in-a-/10206835.

Should I go ahead and publish this now?

WRT the two reports that I need to generate for the WinForms-Blazor app (Dock > Fill and `Counter` component squiggle), I'll do that first thing Saturday morning, as I'm about to go OOF again in a few minutes. I'll send those two links via email tomorrow morning.

I'm about to place the link to the Java SDK Dev Community issue into the email chain.

**Review by danroth27 — APPROVED** [review]
LGTM

**Pull request merged**